src/modules/docker_context.rs

```diff
@@ -0,0 +1,48 @@
+use dirs::home_dir;
+
+use super::{Context, Module, RootModuleConfig};
+
+use crate::configs::docker_context::DockerContextConfig;
+use crate::utils;
+
+const DOCKER_CONFIG_FILE: &str = ".docker/config.json";
+
+/// Creates a module with the currently active Docker context
+///
+/// Will display the Docker context if the following criteria are met:
+///     - There is a file named `$HOME/.docker/config.json`
+///     - The file is JSON and contains a field named `currentContext`
+///     - The value of `currentContext` is not `default`
+pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
+    let mut module = context.new_module(DOCKER_CONFIG_FILE);
+    let config: DockerContextConfig = DockerContextConfig::try_load(module.config);
+
+    if config.only_with_compose_yml
+        && !context
+            .try_begin_scan()?
+            .set_files(&["docker-compose.yml"])
```

Comment:
Would it be useful to also enable the module if the dir has a `Dockerfile` too?
```suggestion
            .set_files(&["docker-compose.yml", "Dockerfile"])
```

If you make this change then maybe it would be better to change the `only_with_compose_yml` config var to something more generic maybe `only_with_docker_files` or `only_with_files`?